.github/workflows/cicd-pipeline.yml

```diff
@@ -190,7 +191,7 @@ jobs:
       - name: Run contract tests
         env:
           NAMESPACE: erica-staging
-          IMAGE_TAG: 460
```

Comment:
Why did used a constant for the image tag?

Reply:
Good question. The constant was already set before. I think Julian set it. But yes we always need the last one